pkg/master/master.go

```diff
@@ -253,6 +255,47 @@ func (m *Master) InstallAPIs(c *Config) {
 		}
 		allGroups = append(allGroups, group)
 	}
+	// Install batch unless disabled.
+	if !m.ApiGroupVersionOverrides["batch/v1"].Disable {
```

Comment:
How about using job's v1.SchemeGroupVersion.String() instead of above string to remove repeating the strings?